src/deskolem.ml

```diff
@@ -246,180 +287,135 @@ let get_y : term -> term Bindlib.var * term = fun t ->
     |Symb({sym_name = "∃"; _})     ->
```

Comment:
This is not correct to test symbol name equality. You should use Term.is_symb cfg.symb_... instead.